integrations/docker/images/base/chip-build/version

```diff
@@ -1 +1 @@
-43 : [NuttX] Fix build errors
```

Comment:
This has to be in a separate PR:

- one PR to change Dockerfiles + chip-build version
- a separate PR to make the changes in syscfg/readmes etc.